astroid/brain/brain_gi.py

```diff
@@ -29,6 +29,34 @@
 
 _identifier_re = r"^[A-Za-z_]\w*$"
 
+_special_methods = [
```

Comment:
This should be a frozenset:
```suggestion
_special_methods = frozenset({
```